packages/server/src/events/tasks.ts

```diff
@@ -1,8 +1,26 @@
+import { PrismaClientSingleton } from '../utils';
+
 /**
  * Task ( Runs after the article is published successfully ).
  * Create the article story
  */
-export async function CreateArticleStory(email: string, articleId: number) {}
```

Comment:
email attribute should be used, doing so would improve the security. But it is good for now.